tmt/steps/provision/artemis.py

```diff
@@ -508,9 +511,12 @@ def options(cls, how: Optional[str] = None) -> List[tmt.options.ClickOptionDecor
                 help=f'A factor for exponential API retry backoff, '
                      f'{DEFAULT_RETRY_BACKOFF_FACTOR} by default.',
                 ),
-            ] + super().options(how)
+            ]
+        return options
 
-    # FIXME: ignore - https://github.com/teemtee/tmt/issues/1437
+    # More specific type is a violation of Liskov substitution principle, and mypy
+    # complains about it - rightfully so. Ignoring the issue which should be resolved
+    # with https://github.com/teemtee/tmt/pull/1439.
```

Comment:
The `FIXME` comment was correct, probably changed recently during the post-annotations review.